test/lru_cache_test.rb

```diff
@@ -0,0 +1,89 @@
+require 'test_helper'
+
+class TestLRUCache < Minitest::Test
```

Comment:
Several of the tests aren't passing for me?

Reply:
Should be fixed now! 🎉